modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/rest-openapi.yaml

```diff
@@ -464,6 +464,136 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/Keyword"
             tags: ["Keyword"]
+    "/asset-libraries/{assetLibraryId}/keywords/by-external-reference-code/{externalReferenceCode}":
+        delete:
+            # @review
+            description:
+                Deletes the asset library's keyword by external reference code.
+            operationId: deleteAssetLibraryKeywordByExternalReferenceCode
+            parameters:
+                - in: path
+                  name: assetLibraryId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+            responses:
+                204:
+                    content:
+                        application/json: {}
+                        application/xml: {}
+            tags: ["Keyword"]
+        get:
+            # @review
+            description:
+                Retrieves the asset library's keyword by external reference code.
+            operationId: getAssetLibraryKeywordByExternalReferenceCode
+            parameters:
+                - in: path
+                  name: assetLibraryId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+                - in: query
+                  name: fields
+                  schema:
+                      type: string
+                - in: query
+                  name: restrictFields
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Keyword"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Keyword"
+            tags: ["Keyword"]
+        post:
```

Comment:
The pattern in the REST APIs with ERC is having just PUT endpoints by ERC. So with the PUT method, we perform an UPSERT operation by its ERC. That is, if the element does not exist, it is created; otherwise it is updated.

Reply:
thanks!

I will remove the post and rebuild this